components/cdc/src/endpoint.rs

```diff
@@ -595,6 +601,19 @@ impl<T: 'static + RaftStoreRouter<E>, E: KvEngine> Endpoint<T, E> {
             return;
         }
 
+        if (kv_api == ChangeDataRequestKvApi::RawKv && api_version != ApiVersion::V2)
+            || (kv_api == ChangeDataRequestKvApi::TxnKv)
+        {
+            error!("RawKv are supported by api-version 2 only, TxnKv aren't supported now.");
```

Comment:
```suggestion
            error!("cdc RawKv are supported by api-version 2 only, TxnKv aren't supported now.");
```

Please always add `"cdc"` prefix for every log.